Brightness Filter

Seif-Ayman opened the pull request:
Add brightness filter with 8 levels, 4 To increase brightness, 4 to lower brightness.
add 2 options, to apply all 8 levels or to apply only 1 filter with comparasion in using matplotlib plots.

Seif-Ayman committed:
Added brightness filter with 8 levels,
4 To increase brightness, 4 to lower brightness.
added 2 options, to apply all 8 levels or to apply only 1 filter with comparasion in using matplotlib plots.

Ozzy-ZY commented:
please restructure the code to be easily maintained and logically separated @Seif-Ayman

Seif-Ayman committed:
Add Brightness filter, Grayscale filter.
Brightness fitler has 8 levels:
-4: "Very dark (20% brightness)",
-3: "Darker (30% brightness)",
-2: "Slightly dark (50% brightness)",
-1: "Lil dark (70% brightness)",
0: "Normal (100% brightness)",
1: "Slightly bright (130% brightness)",
2: "Somewhat bright (160% brightness)",
3: "Brighter (190% brightness)",
4: "Very bright (220% brightness)".

Grayscalefilter uses weighted channels to apply the filter.

Add function to save images in specified path (images folder), display comparsion using Plots in matplot library, and another function to check if the directory exists, if not it will create a new directory.

Add a simple menu in the CLI that contains options to Load, Apply filters and exit the program.
